pages/dc.js

```diff
@@ -0,0 +1,805 @@
+          }}
+        >
+          {Object.entries({
+            'Who can participate in Scrapyard?': (
+              <>
+                All high-school & upper-middle-school aged students are welcome
+                to come! You don't have to be a member of the Hack Club
+                community or be a Hack Club leader.
+              </>
+            ),
+            'All this, for free?': (
+              <>
+                Yep! Food, swag and good vibes are all included. Plus, if you’re
+                joining us from afar,{' '}
+                <Link href="https://gas.hackclub.com/">
+                  we’ll cover the cost of gas or a bus / train ticket
+                </Link>
+                .
+              </>
+            ),
+            'What do I need?': (
+              <>
+                Your laptop, chargers, and an open mind! If you're going to an
+                overnight event, bring toiletries and sleeping bagstoo.
+                Additionally, if you plan to work on a hardware project, bring
+                the tools you'll need.
+              </>
+            ),
+            'I’m not good at coding. Can I still participate?': (
+              <>
+                This hackathon is for creatives of all skill levels! We'll have
+                workshops and other events so join us and let's learn together.
+                If you'd like to start exploring some introductory projects,
+                check out Hack Club Workshops.
+              </>
+            ),
+            'What can I make at Scrapyard?': (
+              <>
+                The scrappiest thing you can imagine –- jank is encouraged.
+                Games? Apps? Websites? Programming languages? <em>Hardware?</em>{' '}
+                You name it! We’ll have a bunch of resources and mentors to help
+                you out.
+              </>
+            ),
+            'What has Hack Club done before?': (
+              <>
+                Hack Club has run an{' '}
+                <Link href="https://youtu.be/PnK4gzO6S3Q" target="_blank">
+                  overnight hackathon
+                </Link>{' '}
+                in San Francisco, a{' '}
+                <Link
+                  href="https://www.youtube.com/watch?v=H5RPsCMl3uM"
+                  target="_blank"
+                >
+                  Game Jam
+                </Link>{' '}
+                across 50 cities, a hackathon on a{' '}
+                <Link href="https://youtu.be/2BID8_pGuqA" target="_blank">
+                  Train
+                </Link>{' '}
+                from Vermont to Los Angeles, and much more!
+              </>
+            ),
+            'What if my parents are concerned?': (
+              <>
+                We’re here to help! Our parents guide will be released soon, but
+                they can reach out to us at{' '}
+                <Link href="mailto:dc@scrapyard.hackclub.com">
+                  scrapyard@hackclub.com
```

Comment:
Change the displayed text to dc@scrapyard.hackclub.com here and in the FAQ below it